extensions/vscode/src/watchers.ts

```diff
@@ -89,28 +90,41 @@ export class ConfigWatcherManager implements Disposable {
   pythonPackageFile: FileSystemWatcher | undefined;
   rPackageFile: FileSystemWatcher | undefined;
 
-  constructor(cfg?: Configuration) {
+  constructor(
+    contentRecord: ContentRecord | PreContentRecord,
+    cfg: Configuration,
+  ) {
     const root = workspace.workspaceFolders?.[0];
-    if (root === undefined || cfg === undefined) {
+    if (root === undefined) {
       return;
     }
-
+    const configurationFileUri = Uri.joinPath(
+      root.uri,
+      contentRecord.projectDir,
+      ".posit",
+      "publisher",
+      contentRecord.configurationName,
```

Comment:
Can you clarify why this changed from `cfg.configurationPath`? Also, does it need a `.toml` suffix?

Reply:
Yeah I'm a bit confused by this as well. We have the `configurationPath` why not just use that as before with `new RelativePattern` and without making our own path?

Reply:
I'll check, but I think this was probably just an oversight. I'll mark this and the other as being switched back and will test it from there. I'm sure the both of you are probably right.